receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md

```diff
@@ -14,6 +14,7 @@ These are the metrics available for this scraper.
 | **process.disk.io.write** | Disk bytes written. | By | Sum(Int) | <ul> </ul> |
 | **process.memory.physical_usage** | The amount of physical memory in use. | By | Sum(Int) | <ul> </ul> |
 | **process.memory.virtual_usage** | Virtual memory size. | By | Sum(Int) | <ul> </ul> |
+| **process.paging.faults** | Number of page faults the process has made. This metric is only available on Linux. | {faults} | Sum(Int) | <ul> <li>type</li> </ul> |
```

Comment:
Do you think this metric is important enough to be added as enabled by default? Maybe better to set it as optional? That would be a non-disruptive change

Reply:
No problem at all making it optional, I'll make this change.